### DISCUSSION TITLE
Fitted values from AutoETS or AutoARIMA estimation

### DISCUSSION THREAD

**Discussion by AFilipaVEL** [Nixtla/statsforecast#799] [General]
Hi all,

Is it already available the method for obtainning the **fitted values** after estimating an AutoETS or an AutoARIMA model, based on a **spark** dataframe? If so, how can i proceed to get those?

Thanks in advance!

**Comment by jmoralez** [comment]
Hey. You should be able to get them as in the local interface, e.g.
```python
sf = StatsForecast(...)
forecasts = sf.forecast(..., fitted=True)
in_sample = sf.forecast_fitted_values()
```

**Comment by AFilipaVEL** [comment]
It works!! 😄 Thanks a lot!
